src/SFA.DAS.ApprenticeCommitments.Web/Pages/Apprenticeships/YourApprenticeshipDetails.cshtml

```diff
@@ -0,0 +1,97 @@
+﻿@page "/apprenticeships/{apprenticeshipId}/yourapprenticeshipdetails"
+@model SFA.DAS.ApprenticeCommitments.Web.Pages.Apprenticeships.YourApprenticeshipDetails
+@{
+    ViewData["Title"] = "Your Apprenticeship Details";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        <partial name="ValidationSummary" />
+
+        <h1 asp-hide="@Model.ConfirmedApprenticeshipDetails" class="govuk-heading-xl">@ViewData["Title"] </h1>
+
+        <div asp-show="@Model.ConfirmedApprenticeshipDetails" class="app-notification-banner app-notification-banner--with-icon app-notification-banner--success">
+            <span class="app-notification-banner__icon das-text--success-icon"></span>
+            You have confirmed your apprenticeship details
+        </div>
+
+        <div class="govuk-body">
+
+            <table class="govuk-table data-table">
+                <tbody class="govuk-table__body">
+                    <tr class="govuk-table__row">
+                        <th scope="row" class="govuk-table__header">Apprenticeship</th>
+                        <td class="govuk-table__cell">@Model.CourseName</td>
+                    </tr>
+                    <tr class="govuk-table__row">
+                        <th scope="row" class="govuk-table__header">Level</th>
+                        <td class="govuk-table__cell">@Model.CourseLevel</td>
+                    </tr>
+                    @if (Model.CourseOption != null)
+                    {
+                        <tr class="govuk-table__row">

```

Comment:
As an alternative, instead of the `@if` we could use `asp-show(Model.CourseOption)` on the `tr`.  Which do you prefer?

Reply:
asp-show only seems to work with type bool, I'm getting an error as it's a string, would I need to create a new taghelper?

Reply:
It does look cleaner that way and works with now, I've changed it